modules/builds-configmaps-use-case-setting-command-line-arguments.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-setting-command-line-arguments_{context}"]
+= Setting command-line arguments with ConfigMaps
```

Comment:
I think it might be useful to include a reference to container commands in this title, for findability / context. For example:

 = Setting command-line arguments for container commands with ConfigMaps